src/core/components/accordion/accordionItemContent/accordionItemContent.tsx

```diff
@@ -0,0 +1,27 @@
+import { AccordionContent as RadixAccordionContent } from '@radix-ui/react-accordion';
+import classNames from 'classnames';
+import { forwardRef, type ComponentPropsWithRef } from 'react';
+import './index.css';
```

Comment:
We currently have `index.css` files to handle stylesheet imports, you can update the `/components/index.css` file to add `@import "./accordion"` import and create a `/components/accordion/index.css` file that imports the accordion-item-content style. With this we don't need to import the style on components and you can remove the css mock for Jest. This should also fix the build. You can check how it's currently done for dialogs